main.go

```diff
@@ -77,4 +79,7 @@ func main() {
 	// logs.SetLevel(logs.LevelInformational)
 	logs.SetLogFuncCall(false)
 	beego.Run(fmt.Sprintf(":%v", port))
+
+	// Execute main.go again to run the ladpserver
+	controllers.StartLdapServer()
```

Comment:
Will running again work? Why? Why not run `go controllers.StartLdapServer()` before `beego.run()`?